harvester/harvester
# Bump longhorn to v1.1.2

**Problem:**
Bump longhorn to v1.1.2 and verify the Harvester is full compatible with the latest longhorn updates.

**Solution:**
Bump longhorn to v1.1.2

**Related Issue:**
https://github.com/harvester/harvester/issues/1016, https://github.com/harvester/harvester/issues/849

**Test plan:**
1. verify all the pods in the `longhorn-system` are healthy.
2. verify uploading VM images is working as expected.
3. verify creating VMs using the longhorn storage driver is working as expected.
